### ISSUE TITLE
Add proper checks for input format

### ISSUE THREAD

**Issue by adityabisoi** [adityabisoi/notation-converter#50]
Add checks to ensure the input format is correct for each operation
Ex: infix input if entered in a postfix-infix transition, must give out an error.

**Comment by Gurdeep475** [comment]
I would like to work on this issue under GSSOC`22. Please assign it to me.

**Comment by Gurdeep475** [comment]
Thanks. I am working on it.

**Comment by Shweta597** [comment]
I would like to work on this issue under GSSOC'22. Please assign it to me.

**Comment by adityabisoi** [comment]
@Gurdeep475 Your status?

**Comment by Gurdeep475** [comment]
working on it.

**Comment by Gurdeep475** [comment]
@adityabisoi how should I give an error, like should I create an alert that displays an invalid expression message or simply throw the error?

**Comment by adityabisoi** [comment]
> @adityabisoi how should I give an error, like should I create an alert that displays an invalid expression message or simply throw the error?

Show it to the user.

**Comment by adityabisoi** [comment]
@Gurdeep475 Updates on this issue?

**Comment by Gurdeep475** [comment]
@adityabisoi yes, most of the work is done, just working on one remaining algo.

**Comment by Gurdeep475** [comment]
@adityabisoi need some guidance here, I have got the algorithms for validating prefix notation but am not able to get the same for infix and postfix ones.

**Comment by SonakshiGoyal** [comment]
I would like to work on this issue under GSSOC`22. Please assign it to me.

**Comment by adityabisoi** [comment]
@Gurdeep475 https://stackoverflow.com/questions/39823568/how-can-i-check-whether-the-given-expression-is-an-infix-expression-postfix-exp

**Comment by adityabisoi** [comment]
@SonakshiGoyal I shall assign this issue to you if @Gurdeep475 doesn't make a PR until tomorrow.

**Comment by adityabisoi** [comment]
This issue is now open to contributions.

**Comment by LakshayN** [comment]
@adityabisoi I would like to work on this issue in GSSOC'22.
